packages/fondue/src/components/RichTextEditor/components/Toolbar/FloatingToolbar.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Toolbar tabbing doesn't work as expected. The toolbar disappears immediately when tab is pressed which didn't happen before

Reply:
This one I did not fix because to me the behavior looks the same as the old one? Let me know what I missed. Either way, none of them were accessible, which is not ideal although I'd prefer not to tackle a18y in this already monstrous PR.